components/match2/commons/match_group_input_util.lua

```diff
@@ -1208,7 +1208,11 @@ end
 ---@field getExtraData? fun(match: table, game: table, opponents: table[]): table?
 ---@field getMapName? fun(game: table): string?
 ---@field getPlayersOfMapOpponent? fun(game: table, opponent:table, opponentIndex: integer): table[]
+---@field getPatch? fun(game: table): string?
+---@field getMode? fun(game: table, opponenets: table[]): string?
```

Comment:
Mode is already available on main, rebase?

Reply:
ugh, didn't see, but it comes too early for it to be usable for sc2 (need game.opponents already set), will have to check if moving it down would hurt the other wiki it is used on

Reply:
> ugh, didn't see, but it comes too early for it to be usable for sc2 (need game.opponents already set), will have to check if moving it down would hurt the other wiki it is used on

Only used on magic atm, there was 2 more  none-craft wikis that needed it as well, but don't have them on the top of my head)